modifier.go

```diff
@@ -2,83 +2,100 @@ package main
 
 import (
 	"bufio"
+	"flag"
 	"fmt"
 	"log"
 	"os"
 	"regexp"
 	"strconv"
 	"strings"
+	"sync"
+	"time"
+
+	"github.com/markhaur/srt-time-modifier/config"
+	"github.com/pkg/errors"
 )
 
-func main() {
+var regexPattern = regexp.MustCompile("[0-9]{2}:[0-9]{2}:[0-9]{2},[0-9]{3} --> [0-9]{2}:[0-9]{2}:[0-9]{2},[0-9]{3}")
 
-	if len(os.Args) < 4 {
-		fmt.Println("Usage: go run modifier.go <inputfile> <outputfile> <modify value>")
-		return
+func main() {
+	var configPath string
+	var duration time.Duration
+	var inputFile string
+	var outputFile string
+	flag.StringVar(&configPath, "config", "./config.yml", "Path to configuration file.")
+	flag.DurationVar(&duration, "duration", 0, "Time modify duration. Must be negative for now.")
+	flag.StringVar(&inputFile, "i", "./input/Hz.Omer.S01E01.srt", "Path to input file.")
+	flag.StringVar(&outputFile, "o", "./output/Hz.Omer.S01E01.srt", "Path to output file.")
```

Comment:
these should have empty defaults i think
```suggestion
	flag.StringVar(&inputFile, "i", "", "Path to input file.")
	flag.StringVar(&outputFile, "o", "", "Path to output file.")
```
otherwise a user can never load from a config file

Reply:
great!